pages/repo/edit.js

```diff
@@ -772,14 +778,20 @@ export class EditorPage extends Component {
       <Frame raw>
         <Frame.Header
           barStyle={{
-            borderBottom: activeUsers.length
+            borderBottom: isTemplate
+              ? `3px solid ${colors.divider}`
+              : activeUsers.length
```

Comment:
On second though, worrying about suppressing active user notice, I'd prefer a `background-color: colors.secondaryBg`, which we could also add to `pages/index.js` when in templates mode